pkg/eventing/v1beta1/client.go

```diff
@@ -310,6 +314,10 @@ type BrokerBuilder struct {
 // NewBrokerBuilder for building broker object
 func NewBrokerBuilder(name string) *BrokerBuilder {
 	return &BrokerBuilder{broker: &v1beta1.Broker{
+		TypeMeta: meta_v1.TypeMeta{
```

Comment:
you could use `updateEventingGVK()` here, so that we do update always the same way.

Reply:
good point. actually, I would consider an error in this call as being fatal, so I wonder whether we could bail out here with a fatal exception. alternative, ignore an error here as it might no happen (we are calling the function with no user output, just constants that should have been registered. This is probably the easiest way.

The benefit is that when eventing is moving to `v1` we only have a single spot to update (`v1` will come soon).

Reply:
@rhuss force pushed an update to use `updateEventingGVK()`